### PR TITLE
6.3

### PR DESCRIPTION
**Pull request by erinc35**
@bontgoy

Could you please check my code? I think I did what the instructions said but I need your opinion. Thanx a lot.

### CODE REVIEW

File: ruby/santa.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+class Santa
+   attr_reader :age, :ethnicity
+   attr_accessor:gender
```
Comment (Bontgoy) [inline ruby/santa.rb]: needs a space `attr_accessor:gender` should be `attr_accessor :gender`

File: ruby/santa.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    @reindeer_ranking = ["Rudolph", "Dasher", "Dancer", "Prancer", "Vixen", "Comet", "Cupid", "Donner", "Blitzen"]
+    @age = rand(140)
+  end
+
+  def speak
+    puts "Ho, ho, ho! Haaaapy holidays!"
+  end
+
+  def eat_milk_and_cookies(cookie)
+    puts "That was a good #{cookie}."
```
Comment (Bontgoy) [inline ruby/santa.rb]: careful with puts as the return.  You should return the string and let the user interface print it out if you want to see it.

File: ruby/santa.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  end
+
+  def celebrate_birthday
+  	@age += 1
+  end
+
+  def get_mad_at(reindeer)
+  	@reindeer_ranking.delete(reindeer)
```
Comment (Bontgoy) [inline ruby/santa.rb]: refactor for single call

```
@reindeer_ranking.push(@reindeer_ranking.delete(reindeer))
```

File: ruby/santa.rb (hunk 2 above)
Comment (erinc35) [inline ruby/santa.rb]: Does that apply to all my methods?

Comment (erinc35) [inline ruby/santa.rb]: I mean , should I create my methods like this:

def speak
    "Ho, ho, ho! Haaaapy holidays!"
  end

  def eat_milk_and_cookies(cookie)
    "That was a good #{cookie}."
  end

without returning and let the user print like

p santa.speak or p santa.eat_milk_and_cookies(cookie) ?

Comment (Bontgoy) [inline ruby/santa.rb]: i think that that is the pest practice.  I do not like me methods returning `nil` unless that is my expected value for some other type of logic.

`puts` is for debugging not for feature code

Comment (erinc35) [inline ruby/santa.rb]: Got it, thanx for the feedback. Really appreciate it.

**Comment by Bontgoy** [comment]
nice fixes :shipit:

**Pull request merged**